src/Common/StringSearcher.h

```diff
@@ -569,8 +569,10 @@ class StringSearcher<true, ASCII> : public StringSearcherBase
     uint8_t first{};
 
 #ifdef __SSE4_1__
-    /// vector filled `first` for determining leftmost position of the first symbol
-    __m128i pattern;
+    uint8_t second{};
+    /// vector filled `first` or `second` for determining leftmost position of the first and second symbols
+    __m128i first_pattern;
+    __m128i second_pattern;
```

Comment:
Unrelated, but a `short` or `uint16_t` would do the job for `cachemask` (l. 578)

Reply:
Done. uint16_t should be fine for cachemask.